packages/flutter_test/lib/src/matchers.dart

```diff
@@ -576,7 +576,9 @@ AsyncMatcher matchesGoldenFile(Object key, {int? version}) {
 ///   final ui.Canvas pictureCanvas = ui.Canvas(recorder);
 ///   pictureCanvas.drawCircle(Offset.zero, 20.0, paint);
 ///   final ui.Picture picture = recorder.endRecording();
+///   addTearDown(picture.dispose);
 ///   ui.Image referenceImage = await picture.toImage(50, 50);
+///   addTearDown(referenceImage.dispose);
```

Comment:
Should these `addTearDown` be removed?

Reply:
Yes, disposing of the `picture` doesn't affect the `referenceImage`.
You can try the following test:
```dart
import 'dart:ui' as ui;
...
testWidgetsWithLeakTracking('test', (widgetTester) async {
  final ui.Paint paint = ui.Paint()
  ..style = ui.PaintingStyle.stroke
  ..strokeWidth = 1.0
  ..color = const ui.Color.fromARGB(255, 255, 0, 0);
  final ui.PictureRecorder recorder = ui.PictureRecorder();
  final ui.Canvas pictureCanvas = ui.Canvas(recorder);
  pictureCanvas.drawCircle(Offset.zero, 20.0, paint);
  final ui.Picture picture = recorder.endRecording();
  final ui.Image image = await picture.toImage(20, 20);
  picture.dispose();
}, leakTrackingTestConfig: LeakTrackingTestConfig.debugNotDisposed());
```

Reply:
I mean there is `///` in front of it, that means it is commented out. Am I missing something?

Reply:
And on line 579

Reply:
Oh, I got it. This is the sample code in the documentation comment, which I have updated:
https://github.com/flutter/flutter/blob/1a830d22a919552ad07f4494a20f3eb55701fe02/packages/flutter_test/lib/src/matchers.dart#L568-L590